demos/meltdown_ud.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <array>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+
+#ifndef __linux__
```

Comment:
I understand this is here since it ensures `signal.h` is available -- and then it made sense to put the `__aarch64__` check right after -- but I'd prefer if they didn't come in the middle of the `#include` list. Move above the first `#include`?

Reply:
Done.